multi-user-dev-server.js

```diff
@@ -15,61 +16,46 @@ const Express = require("express");
  * @param optionsFromUsername A function that takes a username and returns a
  *                            options for multi-user-dev-server. See sample in
  *                            example/app.js.
+ * @param expireUnusedAfterSeconds Number of seconds after which webpack
+ *                            watchers that are unused will be closed /
+ *                            released. 0 or null == no expiration.
```

Comment:
I am compelled to comment that this style of indentation is silly, and I don't understand why some people prefer it. Indentation should be consistent so that the amount of whitespace tells us something about nesting rather than depending purely on the size of a name!

Of course, this isn't directed at you. You're just correctly matching style.